src/web/mjs/connectors/Kuman5.mjs

```diff
@@ -40,7 +33,7 @@ export default class Kuman5 extends Connector {
 
     async _getChapters(manga) {
```

Comment:
In case of applying the customized chapter IDs in `_getMangas` method, this method no longer needs to be overwritten, the base from template should work

Reply:
*`_getChapters`